src/vivarium_gates_mncnh/components/observers.py

```diff
@@ -318,15 +311,8 @@ def configuration_defaults(self) -> dict[str, Any]:
         return {
             "stratification": {
                 self.get_configuration_name(): {
-                    "exclude": ["age_group"],
-                    "include": [
-                        "child_age_group",
-                        "child_sex",
-                        "cpap_availability",
-                        "antibiotics_availability",
-                        "probiotics_availability",
-                        "delivery_facility_type",
-                    ],
+                    "exclude": [],
```

Comment:
This property is not needed at all, as it's inherited from `Observer`. 

Same for the other instances below.